src/main/kotlin/org/wfanet/measurement/dataprovider/common/ElGamalPublicKeyStore.kt

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.wfanet.measurement.dataprovider.common
+
+import org.wfanet.measurement.api.v2alpha.CombinedPublicKey
+import org.wfanet.measurement.api.v2alpha.ElGamalPublicKey
+
+/**
+ * Interface for depencency-injecting a way to get [ElGamalPublicKey]s from [CombinedPublicKey]s.
```

Comment:
Do we still have a concept of CombinedPublicKey? aren't we going to delete the combinedPublicKeysService?